### ISSUE TITLE
comictagger crashes on start-up

### ISSUE THREAD

**Issue by carif** [comictagger/comictagger#209]
Hi there,

I recently installed comictagger on Linux via:
`pip3 install comictagger`
and it always crashes on start-up. This command installed  comictagger 1.3.0a4 .

I also tried an older version, but it crashes with the same error, too: `pip3 install comictagger==1.3.0a0`

This is the error which pops up in the window:
```
Unhandled exception in app:
Traceback (most recent call last):
  File "/home/myuser/.local/lib/python3.10/site-packages/comictaggerlib/main.py", line 92, in ctmain
    tagger_window = TaggerWindow(opts.file_list, SETTINGS, opts=opts)
  File "/home/myuser/.local/lib/python3.10/site-packages/comictaggerlib/taggerwindow.py", line 131, in __init__
    self.archiveCoverWidget = CoverImageWidget(
  File "/home/myuser/.local/lib/python3.10/site-packages/comictaggerlib/coverimagewidget.py", line 91, in __init__
    self.updateContent()
  File "/home/myuser/.local/lib/python3.10/site-packages/comictaggerlib/coverimagewidget.py", line 198, in updateContent
    self.updateImage()
  File "/home/myuser/.local/lib/python3.10/site-packages/comictaggerlib/coverimagewidget.py", line 203, in updateImage
    self.loadDefault()
  File "/home/myuser/.local/lib/python3.10/site-packages/comictaggerlib/coverimagewidget.py", line 273, in loadDefault
    self.setDisplayPixmap(0, 0)
  File "/home/myuser/.local/lib/python3.10/site-packages/comictaggerlib/coverimagewidget.py", line 323, in setDisplayPixmap
    self.lblImage.move((frame_w - img_w) / 2, (frame_h - img_h) / 2)
TypeError: arguments did not match any overloaded call:
  move(self, QPoint): argument 1 has unexpected type 'float'
  move(self, int, int): argument 1 has unexpected type 'float'

```

The terminal output prints the following:
`Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.`

**Comment by lordwelch** [comment]
This is a duplicate of #208 and will be fixed in the next alpha release

**Comment by lordwelch** [comment]
fixed in #214
